test/prober/prober.go

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2020 The Knative Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package prober
+
+import (
+	"testing"
+	"time"
+
+	"github.com/wavesoftware/go-ensure"
+	"go.uber.org/zap"
+	"knative.dev/eventing/test/lib"
+	"knative.dev/eventing/test/lib/resources"
+)
+
+var (
+	// Version is a version of wathola tool version that is built outside of eventing
```

Comment:
what is wathola?

Reply:
It is a testing tool I used to send and check if all events propagated well.

Please discuss that in https://github.com/knative/eventing/pull/2388/files/3746ec01ec63fe65d420a5fe4105fddfb17d9e7d#r406279058